sandbox/server/index.py

```diff
@@ -0,0 +1,57 @@
+from flask import Flask, jsonify, request, abort
+from .utils import sandbox_unsafe_execute
+from .walker.walk_creator import Walker
+from numpy import ndarray
+import json
+
+app = Flask(__name__)
+
+def bad_request(message, **kwargs):
+    return {'message': message, **kwargs}, 500
+
+
+def generate_racer(code_str, timeout):
+    try:
+        execution_result = sandbox_unsafe_execute(code_str, "make_walker", timeout)
```

Comment:
I can't see `sandbox_unsafe_execute` in utils, only `unsafe_execute`?

Reply:
I was missing the rename!